spin/wallet.go

```diff
@@ -0,0 +1,41 @@
+package spin
+
+import (
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/TRON-US/go-btfs/core/commands/storage/upload/helper"
+	"github.com/TRON-US/go-btfs/core/wallet"
+)
+
+const (
+	period = 5 * time.Minute
+)
+
+type walletWrap struct {
+	*helper.ContextParams
+}
+
+func NewWalletWrap(params *helper.ContextParams) *walletWrap {
+	return &walletWrap{params}
+}
+
+func (wt *walletWrap) UpdateStatus() {
+	tick := time.NewTicker(period)
+	defer tick.Stop()
```

Comment:
should be inside the goroutine.. otherwise it will exit right away after UpdateStatus() ends.